browser/src/control/jsdialog/Widget.TreeView.js

```diff
@@ -102,6 +102,15 @@ function _createRadioButton(parentContainer, treeViewData, builder, entry) {
 	return radioButton;
 }
 
+function _createColorPreviewButton(parentContainer, builder, entry) {
+	var colorPreviewButton = L.DomUtil.create('input', builder.options.cssClass + ' ui-treeview-checkbox', parentContainer);
+	colorPreviewButton.type = 'color-preview';
+    colorPreviewButton.style.setProperty("background-image", "url('" + entry.image + "')");
```

Comment:
indent

Reply:
please use single quotes as in the past - we will switch to it again probably. to keep consistent with other examples here